bin/periphery/src/stats.rs

```diff
@@ -86,20 +112,54 @@ impl StatsClient {
     self.system.refresh_memory();
     self.system.refresh_processes(ProcessesToUpdate::All, true);
     self.disks.refresh();
+    self.network.refresh();
   }
 
   fn refresh_lists(&mut self) {
     self.disks.refresh_list();
+    self.network.refresh_list();
   }
 
   pub fn get_system_stats(&self) -> SystemStats {
     let total_mem = self.system.total_memory();
     let available_mem = self.system.available_memory();
+
+    let mut total_ingress: f64 = 0.0;
+    let mut total_egress: f64 = 0.0; 
+
+    // Fetch network data (Ingress and Egress)
+    let network_usage: Vec<SingleNetworkInterfaceUsage> = self.network
+        .iter()
+        .map(|(interface_name, stats)| {
```

Comment:
rename `stats` -> `network`, makes it clearer what we are looping through at a glance.

Reply:
Done